internal/observability/call.go

```diff
@@ -89,11 +94,12 @@ func (c call) endLogs(elapsed time.Duration, err error, isApplicationError bool)
 	fields = append(fields, zap.Duration("latency", elapsed))
 	fields = append(fields, zap.Bool("successful", err == nil && !isApplicationError))
 	fields = append(fields, c.extract(c.ctx))
-	if isApplicationError {
```

Comment:
It's not clear to me why you made the change here.

Reply:
Because I wanted to log the actual error when it is an application error.